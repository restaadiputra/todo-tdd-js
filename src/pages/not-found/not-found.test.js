import React from 'react';
import NotFound from './not-found';
import { renderWithRouter } from 'test/utils';

test('render without crash', () => {
  const { getByText, container } = renderWithRouter(<NotFound />);
  const homeLink = getByText(/back to home/i);
  expect(homeLink).toBeInTheDocument;
  expect(container.firstChild).toMatchSnapshot();
});
