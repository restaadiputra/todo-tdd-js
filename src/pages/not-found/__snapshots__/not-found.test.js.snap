// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render without crash 1`] = `
<div
  class="max-w-lg mx-auto mt-4 flex flex-col items-center justify-center"
>
  <div
    class="flex justify-center items-center mb-4"
  >
    <svg
      class="exclamation w-6 h-6 text-blue-800 mr-3"
      fill="currentColor"
      viewBox="0 0 20 20"
    >
      <path
        clip-rule="evenodd"
        d="M8.257 3.099c.765-1.36 2.722-1.36 3.486 0l5.58 9.92c.75 1.334-.213 2.98-1.742 2.98H4.42c-1.53 0-2.493-1.646-1.743-2.98l5.58-9.92zM11 13a1 1 0 11-2 0 1 1 0 012 0zm-1-8a1 1 0 00-1 1v3a1 1 0 002 0V6a1 1 0 00-1-1z"
        fill-rule="evenodd"
      />
    </svg>
    <p
      class="text-blue-800 text-lg"
    >
      The page you are trying to access doesn't exists.
    </p>
  </div>
  <a
    aria-current="page"
    class="text-blue-400 font-bold text-lg hover:underline active"
    href="/"
  >
    Back to Home
  </a>
</div>
`;
